manuscript/03_variables.md

```diff
@@ -10,7 +10,7 @@ fun main() {
    var b = "ABC"
    println(a) // 10
    println(b) // ABC
-   // a = 12 nie jest możliwe, ponieważ a jest tylko do odczytu!
+   // a = 12 - ponowne przypisanie nie jest możliwe, ponieważ `a` jest tylko do odczytu!
```

Comment:
Przyjąłem konwencję nie stosowania ` w komentarzach nawet jeśli zmniejsza to czytelność

Reply:
Poprawiłem bezpośrednio na masterze. Także złamałem linię, niestety linie kodu nie mogą mieć więcej niż 62 znaki.